x-pack/plugins/cases/public/components/all_cases/table_filters.tsx

```diff
@@ -222,23 +158,23 @@ const CasesTableFiltersComponent = ({
               onSelectionChange={handleSelectedAssignees}
             />
           ) : null}
-          <FilterPopover
+          <MultiSelectFilter
```

Comment:
I was testing PR and just realized that this is the only filter that we display in the selector view where the corresponding column is not visible.

If we filter by multiple tags there is no way of knowing which case has which.

<details><summary>Does this make sense?</summary>
<img width="1479" alt="Screenshot 2023-11-07 at 11 58 24" src="https://github.com/elastic/kibana/assets/1533137/f188ac9f-7adf-4563-9054-82c9622268ff"></details>

@mdefazio @cnasikas

Reply:
Yes, we decided that a couple of releases ago to reduce the space. The same occurs with the Solution filter. @jcger That reminded me that we did not convert the filter to the new component. Should we do it? (on another PR)

<img width="1174" alt="Screenshot 2023-11-07 at 1 53 30 PM" src="https://github.com/elastic/kibana/assets/7871006/9dfbd58d-890c-4cf4-b5b8-d2e88e2a9d47">